include/rgl/api/core.h

```diff
@@ -526,6 +532,16 @@ RGL_API rgl_status_t rgl_entity_set_id(rgl_entity_t entity, int32_t id);
  */
 RGL_API rgl_status_t rgl_entity_set_intensity_texture(rgl_entity_t entity, rgl_texture_t texture);
 
+/**
+ * Set laser retro value for the given Entity.
+ * The value can be retrieved from `RGL_FIELD_LASER_RETRO_F32` point cloud field.
+ * Default retro for the Entity is zero.
+ * @param entity Entity to modify.
+ * @param retro Laser retro value to set.
+ */
+RGL_API rgl_status_t
+rgl_entity_set_laser_retro(rgl_entity_t entity, float retro);
```

Comment:
Formatting is different than in other API calls - `RGL_API rgl_status_t` is one line before call name.